tests/test_project/tests/test_qa.py

```diff
@@ -267,10 +260,9 @@ def test_qa_call_check_rst_file_clean_failure(self):
             pass
         else:
             self.fail('SystemExit not raised')
-        finally:
-            sys.stdout = sys.__stdout__  # reset redirect
 
-    def test_qa_call_check_rst_file_syntax(self):
+    @redirect_stdout(io.StringIO())
```

Comment:
I think `io.StringIO()` can be the default argument value, so that we don't have to pass it everytime on every call!

Reply:
If I try to give it as default, it is giving error as `ValueError: I/O operation on closed file`

Reply:
Should I use io.StringIO() only, if that's the case I can define it in decorator but then we may not be able to provide different output stream other than StringIO

Reply:
Done